setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="chat_exporter",
-    version="1.2",
+    version="1.2.1.1",
```

Comment:
1.2.1

Reply:
I just had to bump it to .1 to get PyCharm to recognise the update